docs/en/peripherals/dshot.md

```diff
@@ -147,3 +147,21 @@ ERROR [dshot] No data received. If telemetry is setup correctly, try again.
 
 Check manufacturer documentation for confirmation/details.
 :::
+
+## Bidirectional DShot (Telemetry)
+
+<Badge type="tip" text="PX4 v1.16" />
```

Comment:
Is this the correct version for it to be introduced?

Reply:
yup!